test/100-basic-name-resolution.bats

```diff
@@ -53,15 +53,15 @@ load helpers
 	gw=$(echo "$config_a1" | jq -r .network_info.podman1.subnets[0].gateway)
 	create_container "$config_a1"
 	a1_pid=$CONTAINER_NS_PID
-	run_in_container_netns "$a1_pid" "dig" "+short" "aone" "@$gw"
+	run_in_container_netns "$a1_pid" "dig" "+short" "aone" "@$gw" "AAAA"
 	assert "$ip_a1"
 	# Set recursion bit is already set if requested so output must not
 	# contain unexpected warning.
 	assert "$output" !~ "WARNING: recursion requested but not available"
 
-	run_in_container_netns "$a1_pid" "dig" "+short" "google.com" "@$gw"
-	# validate that we get an ipv4
-	assert "$output" =~ "[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+"
+	run_in_container_netns "$a1_pid" "dig" "+short" "google.com" "@$gw" "AAAA"
+	# validate that we got valid ipv6
+	expected_rc=0 run_helper ip_calc -c "$output"
```

Comment:
Why expected_rc=0? That should be the default. Also you have to use `ip_calc -6c` because otherwise it will validate both ipv4 and ipv6 addresses and this test must make sure it is an ipv6

Reply:
My bad was not aware the ex_rc=0 is the default and converted to only ipv6.